shared/lib_battery_dispatch.cpp

```diff
@@ -1153,3 +1153,12 @@ bool byLowestMarginalCost::operator() (grid_point const& a, grid_point const& b)
     return a.MarginalCost() < b.MarginalCost();
 
 }
+
+bool byExportPerKWh::operator() (grid_point const& a, grid_point const& b)
+{
+    if (a.ExportPerKWh() == b.ExportPerKWh())
+    {
+        return a.ExportPerKWh() > b.ExportPerKWh();
```

Comment:
@brtietz , can you explain the logic here? If the two grid points are equal then return false - always?

Reply:
This is a typo. I corrected the tiebreaker to ExportPrice (total paid by utility to discharge at full power) Good catch!